.ci/cico_rhche_compatibility_test.sh

```diff
@@ -127,7 +157,34 @@ if [ $RETURN_CODE != 0 ]; then
   url="${url}/comments"
   job_url="https://ci.centos.org/job/devtools-rh-che-rh-che-compatibility-test-dev.rdu2c.fabric8.io/$BUILD_NUMBER/console"
   message="Periodic compatibility check failed. See more details here: $job_url"
-  curl -X POST -s -L -H "Authorization: token $(echo ${FABRIC8_HUB_TOKEN}|base64 --decode)" $url -d "{\"body\": \"$message\"}"
+  curl -X POST -s -L -u ${GITHUB_AUTH_STRING} $url -d "{\"body\": \"$message\"}"
 fi
 
-exit $RETURN_CODE
+echo $RETURN_CODE > compatibility_status
+}
+
+run_tests_timeout_seconds=${RUN_TEST_TIMEOUT:-900}
+SECONDS=0
+touch compatibility_status
+echo "Starting compatibility test"
+runCompatibilityTest &
+COMPATIBILITY_PID=$!
+COMPATIBILITY_STATUS=$(cat compatibility_status)
+echo "Compatibility test running in background thread $COMPATIBILITY_PID $COMPATIBILITY_STATUS"
+while true; do
```

Comment:
1) Why is this sort of timeout better then timeout set for a job?
2) It seems that SECONDS are not incremented.

Reply:
this is a quality of life improvement. before, when the job got stuck on anything, it was running there until it timed out on jenkins and I believe the timeout was 1 hour.

Reply:
the seconds are indeed incremented, you can see it in the logs as `Tick Tock [time]` message

Reply:
it is an internal bash variable that's automatically being incremented during a script run